src/init.cpp

```diff
@@ -1190,12 +1194,29 @@ bool AppInitLockDataDirectory()
     return true;
 }
 
+#ifndef WIN32
+NODISCARD static bool CreatePidFile()
```

Comment:
If you're moving this function anyway, why not move `GetPidFile` as well, it's only used from `init.cpp` too

Reply:
Fixed.